src/components/Profile/InactiveProposalsStake.tsx

```diff
@@ -24,44 +31,52 @@ function InactiveProposalsStake({ myInactiveStakes }) {
   )
   return (
     <Box heading="Inactive proposals stake" padding={3 * GU}>
-      {myInactiveStakes.map((stake) => {
+      {myInactiveStakes.map((stake, index) => {
         return (
-          <ProposalItem
-            amount={stake.amount}
-            compact={compact}
-            gardenId={stake.proposal.organization.id}
-            proposalId={stake.proposal.id}
-            proposalName={stake.proposal.metadata}
-            selectProposal={handleSelectProposal}
-          />
+          <div
+            key={index}
+            css={`
+              display: flex;
+              align-items: center;
+              margin-bottom: ${1 * GU}px;
+            `}
+          >
```

Comment:
can we convert this to styled component? #prettyPlease

Reply:
It was already there. We can do it in a new PR